packages/design-system/src/components/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Button } from '../Button';
+import Alert from './Alert';
+
+const meta: Meta<typeof Alert> = {
+  component: Alert,
+  parameters: {
+    docs: {
+      controls: { sort: ['alpha'] },
+    },
+  },
+  argTypes: {
+    children: { control: 'text' },
+  },
+  args: {
+    hideIcon: false,
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Alert>;
+
+const AlertTemplate: Story = {
```

Comment:
This (and following examples) is how `Template.bind()` would look now.

Reply:
Nice, I like this format of just having a render key.